src/Simplic.FileStructure.Workflow.UI/ForwardWindow.xaml.cs

```diff
@@ -10,6 +10,12 @@ namespace Simplic.FileStructure.Workflow.UI
     /// </summary>
     public partial class ForwardWindow : DefaultRibbonWindow
     {
+        /// <summary>
+        /// Gets or sets a boolean
+        /// It's true when the forward window will be saved, otherwise always false.
+        /// </summary>
+        public bool IsSave { get; set; }
```

Comment:
This is a property, wich should not be placed above the constructor.
Also the naming is kinda weird. 
And it feels weird that this is even required. Should'nt the save logig be executed in the on save?